data/json/items/gun/9mm.json

```diff
@@ -87,6 +92,195 @@
       }
     ]
   },
+  {
+    "id": "glock_43",
+    "copy-from": "gun_base_9mm_semi_pistol",
+    "looks_like": "glock_19",
+    "type": "GUN",
+    "name": { "str": "Glock 43" },
+    "description": "A 9x19mm Glock of diminutive proportions.  Unfortunately it is incompatible with other 9mm Glock magazines.",
+    "weight": "460 g",
+    "volume": "279 ml",
+    "longest_side": "159 mm",
+    "barrel_length": "87 mm",
+    "price": "570 USD",
+    "material": [ "plastic", "steel" ],
+    "ranged_damage": { "damage_type": "bullet", "amount": -1 },
+    "dispersion": 480,
+    "pocket_data": [
+      {
+        "magazine_well": "250 ml",
+        "pocket_type": "MAGAZINE_WELL",
+        "item_restriction": [
+          "glock43_6"
+        ]
+      }
+    ]
+  },  
+  {
+    "id": "glock_43x",
+    "//:" "This one uses 10 round doublestack mags, which are incompatible with the standard variant",
+    "copy-from": "gun_base_9mm_semi_pistol",
+    "looks_like": "glock_19",
+    "type": "GUN",
+    "name": { "str": "Glock 43x" },
```

Comment:
X should be capitalized, and only the MOS has a rail, the non-MOS doesn't have a rail.